homeassistant/components/sensor/darksky.py

```diff
@@ -19,14 +19,17 @@
 from homeassistant.util import Throttle
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['python-forecastio==1.3.5']
+REQUIREMENTS = ['python-forecastio==1.4.0']
 
 _LOGGER = logging.getLogger(__name__)
 
 CONF_ATTRIBUTION = "Powered by Dark Sky"
 CONF_UNITS = 'units'
 CONF_UPDATE_INTERVAL = 'update_interval'
 CONF_FORECAST = 'forecast'
+CONF_LANG = 'lang'
```

Comment:
If you use `CONF_LANGUAGE = 'language'´ here you are in line with sensor.openweathermap.

Reply:
Good point. I actually thought about it myself, don't know why I didn't.